### ISSUE TITLE
No longer needed if used for dependabot

### ISSUE THREAD

**Issue by fregante** [hrvey/combine-prs-workflow#26]
Dependabot finally added grouped updates, so this workflow is no longer needed 🎉 

https://github.blog/2023-08-24-a-faster-way-to-manage-version-updates-with-dependabot/

**Comment by AloisSeckar** [comment]
Thanks for pointing this feature out. Really saved a lot of overhead. It also solved the issue with congflicting `pacakge-lock.json` files.

However, I still find a good use for this workflow on [my demos-nuxt project](https://github.com/AloisSeckar/demos-nuxt) where I have multiple directories each with separate `package.json` file. So I have Dependabot configured for each sub-direcory and it produces multiple PRs, though grouped.

Or is it possible to configure Dependabot to run only once over all sub-directories❓
